bin/console

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "bundler/setup"
+require "trailblazer/finder"
+
+# You can add fixtures and/or initialization code here to make experimenting
```

Comment:
Do we need those files?

Reply:
Also, if you wait till the new `dsl` version 1.0 is out we could use `Inject() => [...]` instead of the not-extendable `:inject`.

Reply:
We are still in version 0.x we can always change.